app/components/use-auth.js

```diff
@@ -0,0 +1,205 @@
+import useMethods from 'use-methods'
```

Comment:
Okay, now that you've gotten all this working, could you just try importing this from civil-client again, but it doesn't work than just add a comment

// in the future we should be importing this from civil-client

Reply:
Oops. I kept use-auth.js on here by accident when I meant to delete it before doing a pr. That file was not being used by anything so I just deleted it now.

Reply:
You can remove the file, git add ., git commit -m "comment", git push  and then the PR will be updated.